uwblueprint/childrens-aid-society
# created branches model & updated user model

## Notion ticket link

[Create Branches Model](https://www.notion.so/uwblueprintexecs/Create-branches-model-8b734166994642a2aa96f122875de528)



## Implementation description
* created a new branches model to match the schema described in the ticket
* updated the user model to reference the id from the branches table as a foreign key
* generated migrations
* updated the upgrade and downgrade functions in migration file & added Algoma as a branch 



## Steps to test
1. flask db upgrade & downgrade should function without errors
2. in the database, there should be a branches table with an entry for Algoma
3. whenever we insert into the users table, an error should pop up if we add an id for the branch_id column that doesn't exist in the branches table



## What should reviewers focus on?
* 


## Checklist
